packages/botonic-react/src/webchat-app.jsx

```diff
@@ -372,18 +375,19 @@ export class WebchatApp {
 
   destroy() {
     if (this.hubtypeService) this.hubtypeService.destroyPusher()
-    unmountComponentAtNode(this.host)
+    this.reactRoot?.unmount()
     if (this.storage) this.storage.removeItem(this.storageKey)
   }
 
   async render(dest, optionsAtRuntime = {}) {
     onDOMLoaded(async () => {
       const isVisible = await this.resolveWebchatVisibility(optionsAtRuntime)
-      if (isVisible)
-        render(
-          this.getComponent(dest, optionsAtRuntime),
-          this.getReactMountNode(dest)
-        )
+      if (isVisible) {
+        const webchatComponent = this.getComponent(dest, optionsAtRuntime)
+        const container = this.getReactMountNode(dest)
+        this.reactRoot = createRoot(container) // createRoot(container!) if you use TypeScript
```

Comment:
is this comment needed? maybe store the container before and ensure that it exists?

Reply:
We can delete it, it is just a reminder, it is like this on the react website.